app/src/main/java/org/openobservatory/ooniprobe/activity/LogActivity.java

```diff
@@ -0,0 +1,103 @@
+package org.openobservatory.ooniprobe.activity;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+
+import androidx.core.content.FileProvider;
+import androidx.recyclerview.widget.LinearLayoutManager;
+
+import com.google.common.collect.Lists;
+import com.velmurugan.inapplogger.LogType;
```

Comment:
I am confused by the fact that the `app/build.gradle` imports `com.github.aanorbel`'s `inapplogger` while here we seem to be importing the original implementation. Is this intended?

Reply:
i added functionality to the package without changing the package name. 
i had to publish the update under a different package name as the owner has not approved new changes to the package.

Reply:
✔️